app/src/main/java/fr/free/nrw/commons/delete/DeleteTask.java

```diff
@@ -161,16 +161,29 @@ protected void onProgressUpdate (Integer... values){
 
     @Override
     protected void onPostExecute(Boolean result) {
+
+        String media_message=media.getDisplayTitle();
```

Comment:
Please use these Java conventions: https://github.com/commons-app/apps-android-commons/wiki/Code-style
So `media_message` -> `mediaMessage` for instance.

Reply:
Thanks am going to do that.

Reply:
I have worked it out